transport/policy_test.go

```diff
@@ -239,24 +239,24 @@ func TestTokenAwareSimpleStrategyPolicy(t *testing.T) { //nolint:paralleltest //
 }
 
 /*
-	mockTopologyTokenAwareNetworkStrategy creates cluster topology with info about 8 nodes
-	living in two different datacenters.
+mockTopologyTokenAwareNetworkStrategy creates cluster topology with info about 8 nodes
+living in two different datacenters.
 
-	Ring field is populated as follows:
-	ring tokens:            50 100 150 200 250 300 400 500 510
-	corresponding node ids: 1  5   2   1   6   4   8   7   3
+Ring field is populated as follows:
+ring tokens:            50 100 150 200 250 300 400 500 510
+corresponding node ids: 1  5   2   1   6   4   8   7   3
 
-	Datacenter:       waw
-	nodes in rack r1: 1 2
-	nodes in rack r2: 3 4
+Datacenter:       waw
+nodes in rack r1: 1 2
+nodes in rack r2: 3 4
 
-	Datacenter:       her
-	nodes in rack r3: 5 6
-	nodes in rack r4: 7 8
+Datacenter:       her
+nodes in rack r3: 5 6
+nodes in rack r4: 7 8
 
-	Keyspace:         "waw/her"
-	strategy: network topology
-	replication factors: waw: 2 her: 3
+Keyspace:         "waw/her"
+strategy: network topology
+replication factors: waw: 2 her: 3
```

Comment:
Is this intentional? This doesn't need a change.

Reply:
Rest LGTM

Reply:
Indeed this change was not intentional. Thanks for noticing! Fixed.

However, you might want to apply it in a separate pull request anyway, it is a result of running `go fmt` (version 1.19) on the file.